assets/js/util.js

```diff
@@ -49,6 +49,29 @@
 	$("#orderNeededDate").attr("min", dtToday);
 	});
 
+	/**
+	 *  Pre-fill order form based on user selection from the homepage
+	 */
+	$(".button").click(function() {
```

Comment:
although this probably isn't a real issue, I noticed that class="button" already existed and used in other places (most of them are commented out, except for the instagram button in gallery.html). Perhaps we should avoid using this class as a target for this function, let's create an id instead (something like "prefill-button") and then use "#prefill-button" as the target.

Reply:
that's a great point. I ended up using "prefill-button" as a class for the button elements and selecting ".prefill-button" in this function. The choice for class was the ease to listen to the button elements as a whole. I think there is a way to achieve the same effect using id since they would be unique but I could not quickly recall it for jQuery.